k8s/openebs-operator.yaml

```diff
@@ -1,23 +1,27 @@
+# Create OpenEBS namespace
 # Define the Service Account
 # Define the RBAC rules for the Service Account
 # Launch the maya-apiserver ( deployment )
 # Launch the maya-storagemanager ( deameon set )
```

Comment:
@ksatchit - Probably fix the above line as well, though not part of this commit.

Reply:
Since the following lines will anyway be self descriptive. We can probably put a simple note saying openebs control plane components and associated CRs and RBAC rules are applied

Reply:
@kmova , have made the suggested changes.